src/main/java/taskmaster/commands/Commands.java

```diff
@@ -0,0 +1,47 @@
+package taskmaster.commands;
+
+import taskmaster.exception.DukeExceptions;
+import taskmaster.userinterface.UserInterface;
+import taskmaster.util.Storage;
+import taskmaster.util.TaskList;
+
+
+/*
+ * This class encapsulates commands that executed
+ * according to the user's input
+ */
+
+public abstract class Commands {
+    protected String command;
+
+    /**
+     * Constructor for Command Objects.
+     *
+     * @param command - The command the user entered
+     */
+
+    public Commands(String command) {
+        this.command = command;
+    }
+
+
+    /**
+     * Execute the command.
```

Comment:
I think there needs to be a new line between the javadoc summary and the param tag

Reply:
Thanks for reading through my code and helping me find these mistakes and also giving me suggestions Jun Hong! 💯